### ISSUE TITLE
Changes to typescript file results into "Error: There can be only one platform. Destroy the previous one to create a new one."

### ISSUE THREAD

**Issue by divyakumarjain** [angular/angular-cli#1370]
- OS?  Windows 7, 8 or 10. Linux (which distribution). Mac OSX (Yosemite? El Capitan?): Windows 8. But did not test it on other distributions.

- Versions. Please run `ng --version`. If there's nothing outputted, please run in a Terminal: 

**ng --version**

(node:19736) fs: re-evaluating native module sources is not supported. If you are using the graceful-fs module, please update it to a more recent version.
angular-cli: 1.0.0-beta.8
node: 6.2.2
os: win32 x64

**node --version**
v6.2.2

- Repro steps. Was this an app that wasn't created using the CLI? What change did you do on your code? etc.

a. Created new app using --mobile
b. Added following dependencies in package.json
    ```"@angular2-material/button": "^2.0.0-alpha.6",
    "@angular2-material/card": "^2.0.0-alpha.6",
    "@angular2-material/core": "^2.0.0-alpha.6",
    "@angular2-material/icon": "^2.0.0-alpha.6",
    "@angular2-material/list": "^2.0.0-alpha.6",
    "@angular2-material/progress-circle": "^2.0.0-alpha.6",
    "@angular2-material/sidenav": "^2.0.0-alpha.6",
    "@angular2-material/toolbar": "^2.0.0-alpha.6"```

c. Also updated devDepencencies to
`"angular2-universal": "0.104.0"`
d. started using angular2-meterial/sidenav in app.component.ts.
e. Open app url in browser and it works correctly.
f. Modifying app.component.ts file after ```ng serve``` results into error mentioned below in both console and browser.

- The log given by the failure. Normally this include a stack trace and some
   more information.

`The Broccoli Plugin: [AppShellPlugin] failed with:

Error: There can be only one platform. Destroy the previous one to create a new one.
    at new BaseException (C:\Users\divyjain\Documents\GitHub\hybris-store-angular-material2\node_modules\@angular\core\src\facade\exceptions.js:20:23)
    at Object.createPlatform (C:\Users\divyjain\Documents\GitHub\hybris-store-angular-material2\node_modules\@angular\core\src\application_ref.js:79:15)
    at Bootloader.platform (C:\Users\divyjain\Documents\GitHub\hybris-store-angular-material2\node_modules\angular2-universal\dist\node\bootloader.js:53:23)
    at new Bootloader (C:\Users\divyjain\Documents\GitHub\hybris-store-angular-material2\node_modules\angular2-universal\dist\node\bootloader.js:19:33)
    at Function.Bootloader.create (C:\Users\divyjain\Documents\GitHub\hybris-store-angular-material2\node_modules\angular2-universal\dist\node\bootloader.js:26:16)
    at AppShellPlugin.build (C:\Users\divyjain\Documents\GitHub\hybris-store-angular-material2\node_modules\angular2-broccoli-prerender\dist\prerender.js:26:58)
    at C:\Users\divyjain\Documents\GitHub\hybris-store-angular-material2\node_modules\angular2-broccoli-prerender\node_modules\broccoli-caching-writer\index.js:152:21
    at lib$rsvp$$internal$$tryCatch (C:\Users\divyjain\Documents\GitHub\hybris-store-angular-material2\node_modules\angular2-broccoli-prerender\node_modules\broccoli-caching-writer\node_modules\rsvp\dist\rsvp.js:1036:16)
    at lib$rsvp$$internal$$invokeCallback (C:\Users\divyjain\Documents\GitHub\hybris-store-angular-material2\node_modules\angular2-broccoli-prerender\node_modules\broccoli-caching-writer\node_modules\rsvp\dist\rsvp.js:1048:17)
    at lib$rsvp$$internal$$publish (C:\Users\divyjain\Documents\GitHub\hybris-store-angular-material2\node_modules\angular2-broccoli-prerender\node_modules\broccoli-caching-writer\node_modules\rsvp\dist\rsvp.js:1019:11)
    at lib$rsvp$asap$$flush (C:\Users\divyjain\Documents\GitHub\hybris-store-angular-material2\node_modules\angular2-broccoli-prerender\node_modules\broccoli-caching-writer\node_modules\rsvp\dist\rsvp.js:1198:9)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)
    at process._tickCallback (internal/process/next_tick.js:98:9)

The broccoli plugin was instantiated at: 
    at AppShellPlugin.Plugin (C:\Users\divyjain\Documents\GitHub\hybris-store-angular-material2\node_modules\angular2-broccoli-prerender\node_modules\broccoli-caching-writer\node_modules\broccoli-plugin\index.js:10:31)
    at AppShellPlugin.CachingWriter [as constructor] (C:\Users\divyjain\Documents\GitHub\hybris-store-angular-material2\node_modules\angular2-broccoli-prerender\node_modules\broccoli-caching-writer\index.js:21:10)
    at new AppShellPlugin (C:\Users\divyjain\Documents\GitHub\hybris-store-angular-material2\node_modules\angular2-broccoli-prerender\dist\prerender.js:15:16)
    at Angular2App._buildTree (C:\Users\divyjain\Documents\GitHub\hybris-store-angular-material2\node_modules\angular-cli\lib\broccoli\angular2-app.js:153:48)
    at new Angular2App (C:\Users\divyjain\Documents\GitHub\hybris-store-angular-material2\node_modules\angular-cli\lib\broccoli\angular2-app.js:53:23)
    at module.exports (C:\Users\divyjain\Documents\GitHub\hybris-store-angular-material2\angular-cli-build.js:10:10)
    at Class.module.exports.Task.extend.setupBroccoliBuilder (C:\Users\divyjain\Documents\GitHub\hybris-store-angular-material2\node_modules\angular-cli\node_modules\ember-cli\lib\models\builder.js:55:19)
    at Class.module.exports.Task.extend.init (C:\Users\divyjain\Documents\GitHub\hybris-store-angular-material2\node_modules\angular-cli\node_modules\ember-cli\lib\models\builder.js:89:10)
    at new Class (C:\Users\divyjain\Documents\GitHub\hybris-store-angular-material2\node_modules\angular-cli\node_modules\ember-cli\node_modules\core-object\core-object.js:18:12)
    at Class.module.exports.Task.extend.run (C:\Users\divyjain\Documents\GitHub\hybris-store-angular-material2\node_modules\angular-cli\node_modules\ember-cli\lib\tasks\serve.js:15:19)
    at C:\Users\divyjain\Documents\GitHub\hybris-store-angular-material2\node_modules\angular-cli\node_modules\ember-cli\lib\commands\serve.js:64:24
    at lib$rsvp$$internal$$tryCatch (C:\Users\divyjain\Documents\GitHub\hybris-store-angular-material2\node_modules\angular-cli\node_modules\ember-cli\node_modules\rsvp\dist\rsvp.js:1036:16)
    at lib$rsvp$$internal$$invokeCallback (C:\Users\divyjain\Documents\GitHub\hybris-store-angular-material2\node_modules\angular-cli\node_modules\ember-cli\node_modules\rsvp\dist\rsvp.js:1048:17)
    at lib$rsvp$$internal$$publish (C:\Users\divyjain\Documents\GitHub\hybris-store-angular-material2\node_modules\angular-cli\node_modules\ember-cli\node_modules\rsvp\dist\rsvp.js:1019:11)
    at lib$rsvp$asap$$flush (C:\Users\divyjain\Documents\GitHub\hybris-store-angular-material2\node_modules\angular-cli\node_modules\ember-cli\node_modules\rsvp\dist\rsvp.js:1198:9)
    at _combinedTickCallback (internal/process/next_tick.js:67:7)`

**Comment by tagazok** [comment]
I have the same issue with angular-cli 1.0.0-beta.10
Except that I only did the ng new projectName --mobile and ng serve, did not even open the code folder :/

**Comment by filipesilva** [comment]
Closed as issue was made obsolete by #1455 - but please tell me if this still happens after the next release.
